Foundation/KVO/TWTKeyValueObserver.h

```diff
@@ -38,7 +41,7 @@ typedef void(^TWTKeyValueObserverChangeBlock)(id observedObject, NSDictionary *c
  */
 @interface TWTKeyValueObserver : NSObject
 
-@property (nonatomic, weak, readonly) id object;
+@property (nonatomic, weak, readonly, nullable) id object;
```

Comment:
Including instances of `target` parameters? Will those get sent up the responder chain?